manifests/l/LightBurnSoftware/LightBurn/1.3.00/LightBurnSoftware.LightBurn.locale.en-US.yaml

```diff
@@ -0,0 +1,27 @@
+# Created with YamlCreate.ps1 v2.1.2 $debug=NVS2.5-1-22610-1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: LightBurnSoftware.LightBurn
+PackageVersion: 1.3.00
+PackageLocale: en-US
+Publisher: LightBurn Software
+PublisherUrl: https://lightburnsoftware.com/
+PublisherSupportUrl: https://forum.lightburnsoftware.com/
+# PrivacyUrl: 
```

Comment:
```suggestion
PrivacyUrl: https://forum.lightburnsoftware.com/privacy
```
Additional metadata

Reply:
I don't think this is correct. That's the privacy policy for the community forum. I can't find one on the main product's website.

Reply:
Ah right, apologies